pkg/apis/networking/types.go

```diff
@@ -476,6 +476,12 @@ type IngressRuleValue struct {
 	// 2. Consider adding fields for ingress-type specific global options
 	// usable by a loadbalancer, like http keep-alive.
 
+	// HTTP is a list of http selectors pointing to backends.
+	// A path is matched against the path of an incoming request. Currently it can
+	// contain characters disallowed from the conventional "path" part of a URL
+	// as defined by RFC 3986. Paths must begin with a '/'.
+	// A backend defines the referenced service endpoint to which the traffic
+	// will be forwarded to.
 	// +optional
 	HTTP *HTTPIngressRuleValue
```

Comment:
So this appears to be copying the doc text from the original `extensions/v1beta1` version of this struct into the modern versions... But those old docs don't seem very good and probably aren't worth copying. I don't think this "documentation" is going to help anyone to use that field...